libs/ui/atoms/src/lib/Icon/Icon.tsx

```diff
@@ -80,126 +80,139 @@ enum IconType {
 }
 
 interface IIconProps {
+  size?: number | undefined;
+  color?: string | undefined;
+  backgroundColor?: string | undefined;
+  borderRadius?: number | undefined;
+  onPress?: void;
   name: IconType;
+  iconStyle?: object | undefined;
 }
 
-const size = 17;
-
 function Icon(props: IIconProps) {
+  const iconProperties = {
+    size: props.size || 20,
+    color: props.color || '#009688',
+    backgroundColor: props.backgroundColor || '##007AFF',
+    borderRadius: props.borderRadius || 5,
```

Comment:
So by default I would have border radius?

Reply:
By default library have 5.

Reply:
> So by default I would have border radius?

By default You have it at the beginning already